lib/matplotlib/axes/_axes.py

```diff
@@ -3050,11 +3050,41 @@ def boxplot(self, x, notch=False, sym=None, vert=True, whis=1.5,
 
         .. plot:: mpl_examples/statistics/boxplot_demo.py
         """
+        if not rcParams['boxplot.whiskers'] == 1.5:
```

Comment:
this probably needs a comment explaining why this is being done to prevent future developers from "fixing" this. Also, why the "not" instead of "!="?